### ISSUE TITLE
Cannot play audio message in safari

### ISSUE THREAD

**Issue by samuvale95** [zhukov/webogram#1844]

**Comment by Ryuno-Ki** [comment]
What does „doesn't work” mean?
Do other browsers work?
Anything on the browser console?

**Comment by jhonklik27** [comment]
Gagal

**Comment by samuvale95** [comment]
Yep with other browser work, the problem is only with safari. When I try to
play audio the there is like a lag on a player and I can't hear nothing

Il ven 3 mag 2019, 21:16 André Jaenisch <notifications@github.com> ha
scritto:

**Comment by YosefSinger** [comment]
Same here, and the issue is across all iphone browsers. They all use webkit.

I purchased a brand new iphone today and it played audio only the first few voice notes. Afterwards can't get it to work.

As reported above, it turns the play button "on" but then it doesn't progress into playing the voice note.

My experience today was very valuable since it showed that there is absolutely no inherent problem (it played the first few voice notes on an out-of-the-box iphone) but something superficial blocked the player from moving along the sound track.

Thanks for all of the awesome features of webogram

**Comment by YosefSinger** [comment]
Seems like all ios devices (iphone, ipad, mac, ipod) just cannot play voice notes(?)

Anyone subscribed to these notifications has a successful safari experience?
Thanks!

**Comment by nimsaraonline** [comment]
I had to face the same here problem. If I use Lumia 540 windows phone's edge browser, voice messages isn't working. but audio messages are working. [see attached image]

Thanks!

![image](https://user-images.githubusercontent.com/49786167/90311477-dab92f80-df18-11ea-818a-916231a29c75.png)
